OpenRA.Mods.Common/Traits/LinkClientManager.cs

```diff
@@ -0,0 +1,372 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Primitives;
+using OpenRA.Support;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Manages LinkClients on the actor.")]
+	public class LinkClientManagerInfo : ConditionalTraitInfo, ILinkClientManagerInfo
+	{
+		[Desc("How long (in ticks) to wait until (re-)checking for a nearby available" + nameof(ILinkHost) + ".")]
```

Comment:
Not sure if you should mention interfaces in modder documentation.

Reply:
I think it's better than excluding it